docs/framework/whats-new/index.md

```diff
@@ -398,9 +403,7 @@ public class StaticResourceResolvedEventArgs : EventArgs
 
 The event is not raised (and its `add` accessor is ignored) unless <xref:System.Windows.Diagnostics.VisualDiagnostics> is enabled and the [`ENABLE_XAML_DIAGNOSTICS_SOURCE_INFO`](xref:System.Windows.Diagnostics.VisualDiagnostics.GetXamlSourceInfo%2A) environment variable is set.
 
-<a name="clickonce472" />
-
-#### ClickOnce
+#### <a name="clickonce472" /> ClickOnce
```

Comment:
The link still isn't working. I think it needs to be on two separate lines as it originally was:

\<a name="clickonce472" />
\#\#\#\# ClickOnce